configs/open-nmt/models/transformer_shared_embedding.py

```diff
@@ -0,0 +1,22 @@
+# taken and copied from https://github.com/OpenNMT/OpenNMT-tf/blob/master/config/models/transformer_shared_embedding.py
+"""A Transformer model sharing all embeddings and softmax weights."""
+
+import tensorflow as tf
+import opennmt as onmt
+
+def model():
+  return onmt.models.Transformer(
+	source_inputter=onmt.inputters.WordEmbedder(
+		vocabulary_file_key="source_words_vocabulary",
+		embedding_size=300),
+	target_inputter=onmt.inputters.WordEmbedder(
+          vocabulary_file_key="target_words_vocabulary",
+          embedding_size=300),
+    num_layers=6,
+    num_units=300,
+    num_heads=6,
```

Comment:
The vanilla Transformer has 8 heads, here you have 6, have you tried 8?

Reply:
Since I'm using Glove embeddings with dimension 300, I have to pick a head size that is a factor of 300. I tried 6 and 10 with slightly better val. score with 6. The original transformer has embeddings of dimension 512, so they could choose 8.